fastapi/applications.py

```diff
@@ -447,6 +448,30 @@ async def read_items():
                 """
             ),
         ] = "/redoc",
+        scalar_url: Annotated[
+            Optional[str],
+            Doc(
+                """
+                The path to the alternative automatic interactive API documentation
+                provided by ReDoc.
```

Comment:
```suggestion
                provided by Scalar.
```

I believe you meant this

Reply:
😅 oh gosh haha i was 4 coffees in when i did this PR, appreciate your review @JacobCoffee and I'll update this ✨